theme/aplus/layout.html

```diff
@@ -48,13 +48,13 @@
     {%- endif %}
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" data-aplus="yes" />
 
-    {#%- for cssfile in css_files %}
-    <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" />
-    {%- endfor %#}
+    {%- for cssfile in css_files %}
+    <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" data-aplus="yes"/>
+    {%- endfor %}
 
-    {#%- for scriptfile in script_files %}
-    <script type="text/javascript" src="{{ pathto(scriptfile, 1) }}"></script>
-    {%- endfor %#}
+    {%- for scriptfile in script_files %}
+    <script type="text/javascript" src="{{ pathto(scriptfile, 1) }}" data-aplus="yes"></script>
```

Comment:
It looks like that `css_files` and `script_files` contain plenty of files included by Sphinx that we don't want to have in A+. That is why those variables used to be commented out. With your change, all those files are included with the `data-aplus` attribute and we can't have that. They might interfere with A+ or some course contents. Some of the JS even crashes on every page load (I can see errors in the browser console). 

Can the template (layout.html) use conf.py variables in if statements in order to avoid the `script_files` variable? If yes, the `include_annotated_js` variable could be used in the template too.

Reply:
If the conf.py variables can not be directly used in the template, then at least the theme settings can be and they are also defined in conf.py (in addition to the `theme/aplus/theme.conf` file that I mentioned in the comments in April). In that case, we would just need to use the theme settings dictionary in conf.py and the annotated.py code would check the flag from there. The `add_javascript_file` and `add_css_file` functions would not be needed anymore.

Reply:
Yes, I will use the theme options. It seems more logic to add those options to the theme than creating isolated variables in the template.